kalite/distributed/static/js/distributed/exercises/models.js

```diff
@@ -13,7 +15,7 @@ var ExerciseParams = {
 };
 
 
-var ExerciseDataModel = Backbone.Model.extend({
+var ExerciseDataModel = ContentModels.ContentDataModel.extend({
```

Comment:
Exercise data now comes from the backend with 'extra_fields' - so it needs the parsing magic of ContentDataModel to inflate this to full JSON.